randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1036,6 +1038,7 @@ Extra - asset_id: collision_camera_010
               All of the following:
                   Lay Cross Bomb
                   Any of the following:
+                      # Movement (Intermediate): https://youtu.be/Dy24YPtX0A8
```

Comment:
The trick that was in there before was meant to be a simple sideways jump with cross bomb midair. So I would prefer if this kind of launch was added as a separate trick.